docs/src/main/sphinx/connector/blackhole.md

```diff
@@ -124,9 +124,11 @@ additional features:
 - {doc}`/sql/drop-table`
 - {doc}`/sql/create-schema`
 - {doc}`/sql/drop-schema`
+- {ref}`sql-view-management`
```

Comment:
`sql-view-management` contains `/sql/alter-view`. Blackhole connector doesn't support it.

Reply:
Thats what I thought from looking at the code.. thanks for confirming. Now I am just not sure about refresh materialized view ..